site2/docs/reference-cli-tools.md

```diff
@@ -335,7 +339,10 @@ Options
 |`-s`, `--subscription-name`|Subscription name||
 |`-t`, `--subscription-type`|The type of the subscription. Possible values: Exclusive, Shared, Failover, Key_Shared.|Exclusive|
 |`-p`, `--subscription-position`|The position of the subscription. Possible values: Latest, Earliest.|Latest|
-
+|`-m`, `--subscription-mode`|Subscription mode.|Durable|
+|`-q`, `--queue-size`|Consumer receiver queue size.|0|
```

Comment:
do you mean "The queue size of the consumer receiver"?

Reply:
changed to `The size of consumer's receiver queue`